.npmignore

```diff
@@ -1,3 +1,2 @@
 hardhat.config.ts
 scripts
-test
```

Comment:
I want to the NPM package to include the tests so we can access some of the helpers and utils we've created along the way.